advisories/github-reviewed/2023/06/GHSA-257q-pv89-v3xv/GHSA-257q-pv89-v3xv.json

```diff
@@ -20,17 +20,12 @@
         "ecosystem": "npm",
         "name": "jquery"
       },
-      "ecosystem_specific": {
```

Comment:
This change was not intended. I used the form, so not sure why this part was deleted

Reply:
It's a tooling quirk. No worries 👍